data/blog/publishing-binaries-on-npm.md

```diff
@@ -0,0 +1,376 @@
+---
+title: 'How to publish binaries on npm'
+date: '2024-1-5'
+tags: ['npm', 'binary', 'cli', 'web']
+draft: false
+summary: 'A comprehensive guide on how to publish binaries on npm without getting fired'
+images: [/images/publishing-binaries-on-npm/hero.jpeg]
+layout: PostLayout
+canonicalUrl: https://sentry.engineering/blog/publishing-binaries-on-npm
+authors: ['lucaforstner']
+---
+
+**This blog post is a comprehensive guide on how to distribute platform-specific binaries over npm.**
```

Comment:
Hmm, I'd maybe rewrite this a bit to a proper introductory sentence or so 🤔 E.g.:

```suggestion
Did you ever have to publish binaries on npm? We do, and it turns out that doing so comes with a lot of potential pitfalls! But don't worry, this blog post can act as a comprehensive guide on how to distribute platform-specific binaries over npm.
```

Reply:
I settled for different wording but fleshed this out a bit. Thanks for the feedback!